client/src/App.js

```diff
@@ -24,32 +24,6 @@ function App() {
   ]);
   const [onCalendar, setOnCalendar] = useState(false);
   const [draggedEvent, setDraggedEvent] = useState()
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
-  const [isLoading, setIsLoading] = useState(true);
-  const [userEmail, setEmail] = useState(null);
-
-  useEffect(() => {
-    axios.get('http://localhost:3000/auth/isLoggedIn', {withCredentials: true})
```

Comment:
This looks like it's removing a bunch of auth code. I'm assuming it moved elsewhere or is being handled another way?

Reply:
Not sure about this one, comes from a different PR I think.